bwi_logging/src/bwi_logging/logger_node.py

```diff
@@ -48,19 +48,21 @@
 from .directory import LoggingDirectory
 
 
+""" Main function. """
 def main(argv=None):
-    """ Main function. """
-    if argv is None:
-        argv = sys.argv
-    if len(argv) < 2:
+
+    # create node for reading ROS parameters
+    rospy.init_node('record')
+
+    # get the topics
+    topics = rospy.get_param('~topics', None)
+    if topics == None or topics == "":
         print('error: no topics to record', file=sys.stderr)
         print("""
 usage: rosrun bwi_logging record topic1 [ topic2 ... ]
 """, file=sys.stderr)
```

Comment:
Looks like we need to update the usage statement to show the ``rosrun`` syntax for setting the ``topic`` parameter.